sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLViewSuite.scala

```diff
@@ -462,4 +487,65 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       }
     }
   }
+
+  test("create a permanent/temp view using a hive function") {
```

Comment:
while it's good to have separate test cases, i find these three separate cases a bit too verbose. why not just put hive, built-in, and permanent into a single query?